src/app/application.h

```diff
@@ -194,6 +199,9 @@ private slots:
 #endif
 
 #ifndef DISABLE_GUI
+    SettingValue<bool> m_storeNotificationTorrentAdded;
```

Comment:
maybe use `CachedSettingValue`? seems the getter function might be called frequently.